README.md

```diff
@@ -1,79 +1,99 @@
+![Github All Releases](https://img.shields.io/github/downloads/gorilla-devs/GDLauncher/total.svg)
 
-[![PRs Welcome](https://img.shields.io/github/license/gorilla-devs/GDLauncher.svg)](http://makeapullrequest.com) [![Travis](https://travis-ci.com/gorilla-devs/GDLauncher.svg?branch=master)](https://travis-ci.com/gorilla-devs/GDLauncher) ![Discord](https://img.shields.io/discord/398091532881756161.svg) ![David](https://img.shields.io/david/gorilla-devs/GDLauncher.svg) ![GitHub code size in bytes](https://img.shields.io/github/languages/code-size/gorilla-devs/GDLauncher.svg)
 
-![](https://img.shields.io/github/release/gorilla-devs/GDLauncher.svg?style=flat)
-![Github All Releases](https://img.shields.io/github/downloads/gorilla-devs/GDLauncher/total.svg)
-[![Read the Docs (version)](https://img.shields.io/readthedocs/gdlauncher/master.svg)](https://gdlauncher.readthedocs.io/en/master/)
+<details>
+ <summary><strong>Table of Contents</strong> (click to expand)
+</summary>
+
+- [GDLauncher](#gdlauncher)
+  - [🎮 What is GDLauncher](#-what-is-gdlauncher)
+  - [⚡️ Comparison with Twitch Launcher](#-comparison-with-twitch-launcher)
+  - [🚀 Getting Started](#-getting-started)
+  - [🎮 Download](#-download)
+  - [🎨 Features](#-features)
+      - [Our features:](#our-features)
+      - [You can also:](#you-can-also)
+      - [Some of the features we are still working on are:](#some-of-the-features-we-are-still-working-on-are)
+  - [💾 Compilation](#-compilation)
+    - [⚙️ Requirements](#%EF%B8%8F-requirements)
+    - [▶️ Steps](#%EF%B8%8F-steps)
+    - [🚚 Packaging](#-packaging)
+  - [🚀 Technologies](#-technologies)
+  - [🎁 Contributing](#-contributing)
+  - [❤️ Author](#-author)
+  - [📜 History](#-history)
+  - [🎓 License](#-license)
+  </details>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, add 2 spaces](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5259002)